README.md

```diff
@@ -1 +1,199 @@
-# mod1-capstone-project
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>My Reggae Capstone</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 Reggae Capstone Project](#-reggae-capstone-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Author ](#-author-)
+      - [👤 Lincoln Gibson](#-lincoln-gibson)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Reggae Capstone <a name="about-project"></a>
+
+This is my first Capstone project that designed to highlight the topics learned, it is build using html, css,javascript and linters.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML/">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>CSS</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>JavaScript</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">CSS</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[A Desktop version]**
+- **[A Mobile version]**
+- **[Button Animation when hover or clicked]**
+- **[Image transition when hovered]**
+- **[The Feature section was dynamically created using JavaScript]**
+- **[The Mobile menu operates using JavaScript]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://lincoln1883.github.io/mod1-capstone-project/)
+
+
+## 🚀 Video Presentation <a name="video-presentation"></a>
+- [Video Presentation](https://www.loom.com/share/59480bb95ab344d19bee283cd3fee8a4)
```

Comment:
- Hey, the video's maximum length should not exceed 5 mins. kindly record another video to ensure you've all the requirements for the video. you can check the project requirement for more details on the video.